backend/sirius-components-collaborative-forms/src/test/java/org/eclipse/sirius/components/collaborative/forms/FormDescriptionAggregatorTests.java

```diff
@@ -0,0 +1,122 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.collaborative.forms;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+import org.eclipse.sirius.components.core.api.IObjectService;
+import org.eclipse.sirius.components.forms.description.FormDescription;
+import org.eclipse.sirius.components.forms.description.GroupDescription;
+import org.eclipse.sirius.components.forms.description.PageDescription;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Test Cases for the {@link FormDescriptionAggregator}.
+ *
+ * @author fbarbin
+ */
+public class FormDescriptionAggregatorTests {
+
+    private static final String LABEL = "Label"; //$NON-NLS-1$
+
+    @Test
+    public void testAggregatorWithMatchingPages() {
+        Object object = new Object();
+        List<FormDescription> formDescriptions = new ArrayList<>();
+
+        FormDescription formDescription = this.createForm(object, 3, 2);
+        FormDescription formDescription2 = this.createForm(object, 3, 1);
+
+        formDescriptions.add(formDescription);
+        formDescriptions.add(formDescription2);
+
+        FormDescriptionAggregator aggregator = new FormDescriptionAggregator();
+        Optional<FormDescription> optional = aggregator.aggregate(formDescriptions, List.of(object), new IObjectService.NoOp());
```

Comment:
Is it relevant to test the behavior of the _FormDescriptionAggregator_ when there are many objects?

Reply:
We would only test that we iterate properly on a for loop somewhere. I wanted to have at least one test here for now.